### ISSUE TITLE
Zotero plugin doesn't open/create notes in Obsidian

### ISSUE THREAD

**Issue by mo-seph** [aidenlx/obsidian-zotero#99]
If I use the Zotero plugin and ask it to create or open notes, it doesn't make anything happen in Obsidian (except a bit of a UI twitch sometimes).

In developer view, I can see it's getting a URL action:
```
{items[0]: '{"key":"JC8HJE4Q","id":5708,"libraryID":1}', type: 'item', version: '0.2.0', action: 'zotero/open'}
```

But nothing seems to happen (currently using latest versions AFAICT.

**Comment by aidenlx** [comment]
try latest zotero and obsidian plugin, it's implemented in v1.0.0

**Comment by mo-seph** [comment]
Works great, thanks!
